File: VB/ChartLesson3/Controllers/ChartController.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports System.Web.Mvc

Namespace ChartLesson3.Controllers
    Public Class ChartController
        Inherits Controller

        Public Function Index() As ActionResult
            Using dbContext = New Models.GspDbContext()
                Return View(dbContext.GSPs.ToList())
            End Using
        End Function
    End Class
End Namespace

File: CS/ChartLesson3/Controllers/ChartController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace ChartLesson3.Controllers {
    public class ChartController : Controller {
        public ActionResult Index() {
            using (var dbContext = new Models.GspDbContext()) {
                return View(dbContext.GSPs.ToList());
            }
        }
    }
}
